N/ngspice/build_tarballs.jl

```diff
@@ -43,4 +43,4 @@ dependencies = Dependency[
 ]
 
 # Build the tarballs, and possibly a `build.jl` as well.
-build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies)
+build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies, preferred_gcc_version=v"8")
```

Comment:
This is required because ngspice started using the C++ duals library which gets misparsed by earlier gcc versions.